src/app/core/admin/admin/admin.component.ts

```diff
@@ -111,8 +111,8 @@ export class AdminComponent implements OnInit {
     const newData = await readFile(this.getFileFromInputEvent(inputEvent));
 
     const dialogRef = this.confirmationDialog.openDialog(
-      $localize`Overwrite complete database?`,
-      $localize`Are you sure you want to restore this backup? This will
+      `Overwrite complete database?`,
```

Comment:
Is this still (and if yes, how) translated now?

And the general reminder: review, discussion and the final changelog / git history are easier if any unrelated things are in separate PRs, even when it feels ridiculous for a few lines of change only ;-)

Reply:
Yes I definitely agree to the second point. Have to be more strict to myself :D 

But no this and also other things that were changed are not translated anymore as they are only internals and confuse people when the translation to a new language is done (at least this was the case for the french translation where I had to tell which ones are irrelevant).